mbuild/tests/test_gmso.py

```diff
@@ -56,13 +56,17 @@ def test_infer_hier(selfs, ethane):
         # Create an ethane box, should be a four structure
         eth_box = mb.packing.fill_box(compound=ethane, n_compounds=1, density=1)
 
+        # TODO: once infer_hierarchy is implemented in gmso, changes these functions
+
         # infer_hierarchy=False
-        unlabeled_top = eth_box.to_gmso(infer_hierarchy=False)
+        # unlabeled_top = eth_box.to_gmso(infer_hierarchy=False)
+        unlabeled_top = eth_box.to_gmso()
```

Comment:
```suggestion
        unlabeled_top = eth_box.to_gmso(parse_label=False)
```

Reply:
Made the change...passes locally, so we'll see on the commit.